EU4toV2/Source/Mappers/ReformMapper.cpp

```diff
@@ -0,0 +1,171 @@
+/*Copyright (c) 2019 The Paradox Game Converters Project
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.*/
+
+
+
+#include "ReformMapper.h"
+#include "Log.h"
+#include "Object.h"
+#include "ParadoxParserUTF8.h"
+#include <vector>
+using namespace std;
+
+
+
+ReformMapper* ReformMapper::instance = nullptr;
+
+
+
+ReformMapper::ReformMapper()
+{
+	LOG(LogLevel::Info) << "Parsing Reform mappings";
+
+	shared_ptr<Object> ReformMapObj = parser_UTF8::doParseFile("ReformMapping.txt");
```

Comment:
Avoid parser_UTF8 and parser_1252 in favor of the new parser. I'm trying to excise them from the converters so people don't need to use Boost.

Reply:
I do not know what new or old parser is.

Reply:
fixed through reduction.